README.md

```diff
@@ -36,7 +43,7 @@ What modules are we using, mainly?
 - [Gulp](http://gulpjs.com/), and;
 - [Sass](http://sass-lang.com/) related modules.
 
-This code is following development rules and styles with [jshint](http://jshint.com/) and [jscs](http://jscs.info/).
+This code is following development rules and styles with [jshint](http://jshint.com/) and [ESLint](http://eslint.org).
 Check commented code to enable integration with Mongo DB via [mongoose](http://mongoosejs.com/).
 
 ## Docker
```

Comment:
@msughawara if you have a specific question about the code, you can comment directly on the line that is problematic.